### PR TITLE
Raise path error for failing read in GetTarFile.

### PR DESCRIPTION
**Pull request by abergmeier**

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: tools/build_defs/docker/utils.py [hunk 1]
```diff
@@ -45,12 +47,16 @@ def GetTarFile(f, name):
   Returns:
     The content of the file, or None if not found.
   """
-  with tarfile.open(f, 'r') as tar:
-    members = [tarinfo.name for tarinfo in tar.getmembers()]
-    for i in ['', './', '/']:
-      if i + name in members:
-        return tar.extractfile(i + name).read()
-    return None
+  with open(f, "rb") as file:
```
Comment (laszlocsomor) [inline tools/build_defs/docker/utils.py]: `file` is also a built-in type, so its use can be confusing in line 52 (it managed to confuse me). Please rename it.

Comment (abergmeier) [reply tools/build_defs/docker/utils.py]: Done.

**Comment by buchgr** [comment]
Was this imported?

**Comment by laszlocsomor** [comment]
Yes, but failed submission. I'll try again. Thanks for the ping!

**Pull request closed without merging**